### PR TITLE
doc_fix

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_sap/issues/7

**This PR will result in the following new package version:** v0.1.1

Non breaking

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

## Bugfix
- Corrects the documentation regarding which schema the models are written to.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [na] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x/ All necessary documentation and version upgrades have been applied
- [na]  docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [ ] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

Just fixed the documentation in the READMe.

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ config-version: 2
 require-dbt-version: [">=1.3.0", "<2.0.0"]
 
 name: 'sap_source'
-version: '0.1.0'
+version: '0.1.1'
```
Comment (fivetran-joemarkiewicz) [inline dbt_project.yml]: We won't be cutting a release so these changes should be reverted back to v0.1.0

File: integration_tests/dbt_project.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: 'sap_source_integration_tests'
-version: '0.1.0'
+version: '0.1.1'
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/dbt_project.yml]: We won't be cutting a release so these changes should be reverted back to v0.1.0

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# dbt_sap_source v0.1.1
+## Bugfix
+- Corrects the documentation regarding which schema the models are written to.
+
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md]: No need for a CHANGELOG entry.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
A few comments to address before approval will be ready

File: CHANGELOG.md (hunk 3 above)
Comment (fivetran-reneeli) [reply CHANGELOG.md]: @fivetran-joemarkiewicz 
Yeah I was wondering about this. Should we keep this note to include in the next release? And if so, should we add a placeholder for the next version. Maybe like v0.1.x. But it's not a significant change so I'm fine removing it entirely.

Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md]: Since this is a fairly small change we can omit the changelog entry.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

**Pull request merged**